lib/main.js

```diff
@@ -79,4 +83,22 @@ module.exports = {
 
 }
 
+function _findFileInParents(path, encoding, silent) {
+  for (var i = 0; i < 100; i++) {
+    try{
+      fs.readFileSync(path, { encoding: encoding })
+      return path
+    }
+    catch (e){
+      path = '../' + path
```

Comment:
What could be the risk with `process.cwd()`? One thing I am not very sure would be how to stop at the root of an OS. 

Thanks for the comments. As I said if we agree on adding this feature I will improve the code. Let me know Max.